src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -133,8 +134,8 @@ public void bindArmControls() {
 	}
 
 	public void bindIntakeControls() {
-		// CommandScheduler.getInstance()
-		// .setDefaultCommand(s.intakeSubsystem, new IntakeDefaultCommand(s.intakeSubsystem));
+		CommandScheduler.getInstance()
+				.setDefaultCommand(s.intakeSubsystem, new IntakeDefaultCommand(s.intakeSubsystem));
```

Comment:
Default commands should not be finish instantly... (see [IntakeDefaultCommand.java#L13-16](https://github.com/robototes/ChargedUp2023/pull/48/files#diff-ee7a5cd617a09a23395b5209f868c5346169298dd7c7cea8786f455751b1df4bR13-R16))